ci/ray_ci/bisect/macos_ci_test.sh

```diff
@@ -0,0 +1 @@
+../macos/macos_ci_test.sh
```

Comment:
need to symlink so this file is in the scope of BUILD.bazel

Reply:
I still feel that this symlink and the genrule is a bit weird..